src/cdk/testing/BUILD.bazel

```diff
@@ -15,15 +14,6 @@ ts_library(
     module_name = "@angular/cdk/testing",
 )
 
-npm_package(
```

Comment:
Why was this removed? I thought it was necessary for cdk/testing to be included in the release?

Reply:
See comment above. We are including `cdk/testing` and `cdk/testing/*` like all other entry-points now. i.e. doing bundling, downleveling etc.

It made configuration easier and also I don't see a benefit in making it inconsistent with what we do for `material/*/testing` entry-points.